doc/source/user_guide/10min.rst

```diff
@@ -16,6 +16,15 @@ Customarily, we import as follows:
    import numpy as np
    import pandas as pd
 
+Basic data structures in pandas
+------------------------------
+
+Information in pandas is organized in two types of classes:
```

Comment:
No big deal, but I'd use `Data` and not `Information`, I think it's more accurate.